app/components/object_storage/policy_templates.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2022 Indoc Research
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+def get_policy_template(dataset_code: str) -> str:
+    """Create policy template for admin user."""
+    template = f'''
+    {{
+        "Version": "2012-10-17",
+        "Statement": [
+            {{
+                "Action": [
+                    "s3:ListBucket",
+                    "s3:GetObject",
+                    "s3:PutObject",
+                    "s3:GetBucketLocation",
+                    "s3:DeleteObject"],
+                "Effect": "Allow",
+                "Resource": ["arn:aws:s3:::{dataset_code}/*"]
+            }}
+        ]
+    }}
+    '''
+    return template
+
+
+TEMPLATES_LIBRARY = get_policy_template
```

Comment:
I suggest to use a dictionary here since library assumes more than one template. Or just use single `get_policy_template` function instead of `TEMPLATES_LIBRARY` var.
```suggestion
TEMPLATES_LIBRARY = {
  'some-name': get_policy_template,
}
```